### ISSUE TITLE
Using types in front

### ISSUE THREAD

**Issue by oscarcalvo** [RobinBuschmann/sequelize-typescript#41]
Hi:

I have defined my types in server side using annotations
`@Table
export default class Car extends Model<Car> { 
  @Column
  name: string;
}`

I am creating a API with node to consume it from front (Angular 2). my question is, are there any way to use Car class in front? of course I dont want to use save method etc etc but I would like to use Car attributtes ('name' in this case) directly in my front without to have to create a dto or something like that.

Thanks.

**Comment by RobinBuschmann** [comment]
Hey @oscarcalvo , thanks for contributing. Unfortunately sequelize-typescript is not working on client-side, because sequelize is not.

But my suggestion is, to define interfaces, which are implemented by your models. These interfaces can also be shared with client.
````typescript
// client + server side :)
interface ICar { name: string }

// server side
@Table 
class Car extends Model<Car> implements ICar { @Column name: string; }
````

**Comment by oscarcalvo** [comment]
Thanks a lot. I was thinking about add logic ( methods) etc as well. 

At the end I have made 
`
export class BaseCar{
    name:string
  public getFullName():string{
    return 'name from BaseCar:'+this.name;
  }
}

@Table
export default class Car extends Model<BaseCar>{ 
  @Column
  name: string; 
  @Column
  surname: string;
}
`
So I can add shared logic to BaseCar and use in front and back, also I can create a BaseCarFront and BaseCarBack and add concrete logic to each one.

To add a new property/field I have to add it in two places (BaseCar, Car)...but its the simplest way I found.

Thanks again.
